api/process_bundles.go

```diff
@@ -0,0 +1,465 @@
+		ErrMalformedBody.Write(w)
+		return
+	}
+
+	census, err := a.db.Census(req.CensusID)
+	if err != nil {
+		ErrGenericInternalServerError.WithErr(err).Write(w)
+		return
+	}
+
+	// Get the user from the request context
+	user, ok := userFromContext(r.Context())
+	if !ok {
+		ErrUnauthorized.Write(w)
+		return
+	}
+
+	// Check if the user has the necessary permissions for the organization
+	if !user.HasRoleFor(census.OrgAddress, db.ManagerRole) && !user.HasRoleFor(census.OrgAddress, db.AdminRole) {
+		ErrUnauthorized.Withf("user is not admin or manager of organization").Write(w)
+		return
+	}
+
+	// generate a new bundle ID
+	bundleID := a.db.NewBundleID()
+	// The cenus root will be the same as the account's public key
+	censusRoot := a.account.PubKey.String()
+
+	if len(req.Processes) == 0 {
+		// Return an error if no processes are provided
+		ErrMalformedBody.Withf("no processes provided").Write(w)
+		return
+	}
+
+	// Collect all processes
+	var processes []internal.HexBytes
+
+	for _, processReq := range req.Processes {
+		if len(processReq) == 0 {
+			ErrMalformedBody.Withf("missing process ID").Write(w)
+			return
+		}
+		processID, err := hex.DecodeString(util.TrimHex(processReq))
+		if err != nil {
+			ErrMalformedBody.Withf("invalid process ID").Write(w)
+			return
+		}
+
+		processes = append(processes, processID)
+	}
+
+	// Find the census participants and get them associated to the bundle
+	orgParticipants, err := a.db.OrgParticipantsMemberships(census.OrgAddress, census.ID.Hex(), bundleID.Hex())
+	if err != nil {
+		ErrGenericInternalServerError.WithErr(err).Write(w)
+		return
+	}
+
+	// if a twofactor census add the process to the twofactor service
+	if census.Type == db.CensusTypeSMSorMail ||
+		census.Type == db.CensusTypeMail ||
+		census.Type == db.CensusTypeSMS {
+		if err := a.twofactor.AddProcess(census.Type, orgParticipants); err != nil {
+			ErrGenericInternalServerError.WithErr(err).Write(w)
+			return
+		}
+	}
+
+	// Create the process bundle
+
+	bundle := &db.ProcessesBundle{
+		ID:         bundleID,
+		Processes:  processes,
+		CensusRoot: censusRoot,
+		OrgAddress: census.OrgAddress,
+		Census:     *census,
+	}
+
+	_, err = a.db.SetProcessBundle(bundle)
+	if err != nil {
+		ErrGenericInternalServerError.WithErr(err).Write(w)
+		return
+	}
+
+	httpWriteJSON(w, CreateProcessBundleResponse{
+		URI:  a.serverURL + "/process/bundle/" + bundleID.Hex(),
+		Root: censusRoot,
+	})
+}
+
+// updateProcessBundleHandler adds additional processes to an existing bundle.
+// Requires Manager/Admin role for the organization that owns the bundle. Returns 200 on success.
+// The processes array in the request must not be empty.
+func (a *API) updateProcessBundleHandler(w http.ResponseWriter, r *http.Request) {
+	bundleIDStr := chi.URLParam(r, "bundleId")
+	if bundleIDStr == "" {
+		ErrMalformedURLParam.Withf("missing bundle ID").Write(w)
+		return
+	}
+
+	bundleID, err := primitive.ObjectIDFromHex(bundleIDStr)
```

Comment:
Nit: I think that this conversion should be placed in `db` package to prevent database logic in the API layer.